classes/components/forms/site/PKPSiteConfigForm.php

```diff
@@ -73,5 +76,17 @@ public function __construct($action, $locales, $site)
             'size' => 'small',
             'value' => $site->getData('minPasswordLength'),
         ]));
+
+        // This parameter is only valuable in a multi-instances installation.
+        if ($numberOfJournals > 1) {
+            $this->addField(new FieldOptions('enableSharedReviewerStatistics', [
+                'label' => __('admin.settings.sharedReviewerStatistics'),
+                'description' => __('admin.settings.sharedReviewerStatistics.description'),
+                'options' => [
+                    ['value' => true, 'label' => __('admin.settings.sharedReviewerStatistics.enable')]
+                ],
+                'value' => $site->getData('enableSharedReviewerStatistics') ? $site->getData('enableSharedReviewerStatistics') : true,
```

Comment:
Not important, but on another place you've used `getData('enableSharedReviewerStatistics') ?? true`, which looks cleaner.